lib/index.js

```diff
@@ -107,27 +124,68 @@ export const metadata = {
   version: require('../package.json').version
 };
 
+//export const pluginConfig = {
```

Comment:
dead code?

Reply:
Ah! I was waiting to rename the curl-endpoint plug in... but actually this plugin doesn't need to import anything from `@bpanel/curl`... so I can just pull this out and leave it as a dep in package.json right?

Reply:
Yep! This would actually probably break since this is just for front end imports which `@bpanel/curl` doesn't have exposed. We don't currently have a way to indicate a dependency on a backend plugin, so peerDeps is our best avenue for at least informing that one is missing.